app/components/NoticeTypeCheckboxes/Notices.ts

```diff
@@ -0,0 +1,134 @@
+export const NoticeTypes = {
```

Comment:
This appears to be copy-pasted from `NoticeTypeCheckboxes`. It is fine to factor this data into a separate file, but please remove one copy or the other.